### ISSUE TITLE
Arathi Basin quest implementation.

### ISSUE THREAD

**Issue by Adwean** [Looking4Group/L4G_Core#3067]
As title states , please add the repeatable quest in Arathi Highlands for Arathi Basin Marks of honor turn in for gold. The WSG version is available the the AB repeatable quest can be done only once. 

http://db.hellground.net/?quest=10535 thats the alliance quest
http://db.hellground.net/?quest=10532 thats the horde quest

Please add them , there are old PvPers with more than 250 Arathi marks. 

Thanks!

**Comment by AnonXS** [comment]
Alliance:

http://wowwiki.wikia.com/wiki/Quest:Arathi_Basin_Resources!

http://wowwiki.wikia.com/wiki/Quest:Arathi_Basin_Resources!_(level_60)

http://db.looking4group.eu/?quest=8155
http://db.looking4group.eu/?quest=8156
http://db.looking4group.eu/?quest=8154
http://db.looking4group.eu/?quest=8080
http://db.looking4group.eu/?quest=8297
http://db.looking4group.eu/?quest=10535

Horde:

http://wowwiki.wikia.com/wiki/Quest:Cut_Arathor_Supply_Lines

http://wowwiki.wikia.com/wiki/Quest:Cut_Arathor_Supply_Lines_(level_60)

http://db.looking4group.eu/?quest=8162
http://db.looking4group.eu/?quest=8161
http://db.looking4group.eu/?quest=8160
http://db.looking4group.eu/?quest=8123
http://db.looking4group.eu/?quest=8299
http://db.looking4group.eu/?quest=10532

--- 

DB wise seems to be not the case, as those quests should make either http://db.looking4group.eu/?quest=8158 or http://db.looking4group.eu/?quest=8165 available and close all other existing arathi quests. Sadly no Questline History on wowhead.

**Comment by Adwean** [comment]
There are 5 available quests from the WSG guy , the max level quest rewards 3g20s , but the 5 quests from the Arathi guy disappears the moment you complete any of the quests, and is not available anymore.

**Comment by AnonXS** [comment]
Yes and i think only the highest Level Quest should be available for PChars.
